lib/task.js

```diff
@@ -11,6 +11,7 @@ import States from './states';
 
 import features from './features';
 import getHostname from './util/hostname';
+import hasPrefixedScopes from './util/scopes';
```

Comment:
I looked into the test failures and this needs to be:
import { hasPrefixedScopes } from './util/scopes';

or in util/scopes.js:14 you make it like this:
export default function hasPrefixedScopes(prefix, resources, taskScopes) {